go.mod

```diff
@@ -33,6 +34,7 @@ require (
 	github.com/ordishs/gocore v1.0.35
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.14.0
+	github.com/rs/zerolog v1.31.0
```

Comment:
I think a `go mod tidy` will remove this

Reply:
updated deps.